docs/source/users/index.md

```diff
@@ -155,12 +155,26 @@ in the SageMaker documentation.
 To use SageMaker's models, you will need to authenticate via
 [boto3](https://github.com/boto/boto3).
 
-For example, to use OpenAI models, install the necessary package, and set an environment
-variable when you start JupyterLab from a terminal:
+For example, to use OpenAI models, use the chat interface settings panel to choose the OpenAI language model:
+
+<img src="../_static/chat-settings-choose-language-model.png"
```

Comment:
The screen shot doesn't show any OpenAI language models. While we as users know that we can scroll down to pick an OpenAI model, in the docs, it might be better to show a screen shot with an OpenAI model visible.

Alternatively, you can change the text above to say, "For example, to use AI21 models, …"